### PR TITLE
UI Work

### PR DESCRIPTION
**Pull request by rares45**
I have redone the UI as in #5.
I also added the about activity.
screenshot1screenshot2

And the rest:

screenshot3screenshot4screenshot5

Btw, I dropped the gesture part. It was too hard to make without 3rd party libraries and it wasn't that intuitive.
I think the only swipe gesture should be to remove the transferView after it is done.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,6 +2,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="slowscript.warpinator">
 
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
+    <uses-permission android:name="android.permission.BLUETOOTH" />
+
```
Comment (rares45) [inline app/src/main/AndroidManifest.xml:10]: Moved them here because Android Studio sugested that

**Comment by rares45** [comment]
@slowscript what do you think?

**Comment by slowscript** [comment]
Hi, it has been a busy few days for me again. Sorry about that. Here's a short review:

I appreciate the time you are putting into this. The design looks great! Only two nitpicks this time:
-The animation is a bit too slow, duration should be same as system animations
-I don't see why the background color needs to be different in light mode. I think most users expect at least the basic color scheme to be standardized across apps.

**Comment by rares45** [comment]
I was literally thinking you might have died. 5 days is a lot. 
About the animation. You are right. What's the duration of system animations?
About the color, I wanted to evidence the transfers panel. To be honest I never tried with a white background. Gonna try and fix the animation.

**Comment by rares45** [comment]
I made the animation last 300 instead of 550. And made the white more white

**Comment by slowscript** [comment]
Thanks. Will merge now.

**Pull request merged**